src/GitHubCreateMergePRs/config.xml

```diff
@@ -8,8 +8,8 @@
     <merge from="release/dev16.2" to="release/dev16.3" />
     <merge from="release/dev16.3" to="release/dev16.4-preview1" />
```

Comment:
are we ready to delete preview1?

Reply:
I wasn't sure - what's the policy on servicing past previews?

Reply:
I think when we close out a preview branch from QB, we keep it around for one cycle just in case there's some emergency servicing thing and then get rid of it

Reply:
I checked with Jason as well, since preview 2 is shipped we can delete